From 33391902abeec66c50b43cfda3da5693a412cb09 Mon Sep 17 00:00:00 2001
From: Bruce Ashfield <bruce.ashfield@windriver.com>
Date: Wed, 20 Jun 2012 21:59:20 -0400
Subject: [PATCH] unionfs: change end_writeback to clear_inode

commit: dbd5768f vfs: Rename end_writeback() to clear_inode()
was merged in 3.5-rc1, so unionfs must be updated to the new naming
scheme.

Signed-off-by: Bruce Ashfield <bruce.ashfield@windriver.com>
---
 fs/unionfs/super.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/fs/unionfs/super.c b/fs/unionfs/super.c
index c3ac814..9cb0551 100644
--- a/fs/unionfs/super.c
+++ b/fs/unionfs/super.c
@@ -835,7 +835,7 @@ static void unionfs_evict_inode(struct inode *inode)
 	struct unionfs_dir_state *rdstate;
 
 	truncate_inode_pages(&inode->i_data, 0);
-	end_writeback(inode);
+	clear_inode(inode);
 
 	list_for_each_safe(pos, n, &UNIONFS_I(inode)->readdircache) {
 		rdstate = list_entry(pos, struct unionfs_dir_state, cache);
-- 
1.7.5.4

